test/azure/generated/AzureCompositeModelClient/models/parameters.ts

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ *
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+ * Changes may cause incorrect behavior and will be lost if the code is
+ * regenerated.
+ */
+
+import * as msRest from "ms-rest-js";
+
+export const subscriptionId0: msRest.OperationURLParameter = {
```

Comment:
Can you sort this list alphabetically? That would definitely help with code reviews when parameters change and it will also help us to compare the differences between `subscriptionId0` and `subscriptionId1`.

Reply:
Yeah, great idea.